blaze-server/src/test/scala/org/http4s/server/blaze/BlazeServerSpec.scala

```diff
@@ -17,6 +21,20 @@ class BlazeServerSpec extends ServerAddressSpec {
 
       true must_== true
     }
+
+    "Startup with a custom ExecutionContext" in {
+      val s = BlazeBuilder
+        .bindAny()
+        .withExecutionContext(
+          ExecutionContext.
+            fromExecutorService(
+              Executors.newFixedThreadPool(5)))
```

Comment:
We need to shut this down, or this will leak threads on test runs.  Not a big deal for a single Travis run, but this will eventually kill a long-running sbt.

Unless we're confirming that code runs on the provided executor, I think we can eliminate this test.  I think this would have passed even with the bug.